src/SprykerSdk/Zed/Integrator/IntegratorConfig.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types = 1);
+
+namespace SprykerSdk\Zed\Integrator;
+
+use Spryker\Shared\Config\Config;
+use Spryker\Shared\Kernel\KernelConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+
+class IntegratorConfig extends AbstractBundleConfig
+{
+    public const MANIFEST_KEY_TARGET = 'target';
+    public const MANIFEST_KEY_SOURCE = 'source';
+    public const MANIFEST_KEY_VALUE = 'value';
+    public const MANIFEST_KEY_DEFAULT_VALUE = 'defaultValue';
```

Comment:
```suggestion
    public const MANIFEST_KEY_DEFAULT = 'default';
```

Instead of relying on casing here, I would prefer we use a simple and speaking value already without inflection issues.